src/components/QueryBuilder.vue

```diff
@@ -103,6 +100,14 @@ export default Vue.extend( {
 				this.$store.dispatch( 'updateProperty', selectedProperty );
 			},
 		},
+		selectedPropertyValueRelation: {
+			get(): string | null {
```

Comment:
why is the return value `string | null`? isn't it always a string? I was assuming the default is = 'matching'? what do you think @micgro42

Reply:
> after taking a second look i think the type should be PropertyValueRelation

I agree

Reply:
@guergana good catch :+1: